07-19/react-props_greeting/src/App.js

```diff
@@ -0,0 +1,14 @@
+import "./styles.css";
+
+export default function App() {
+  return (
+    <>
+      <Greeting name="Leon" />
+      <Greeting isCoach />
```

Comment:
I'd put the name back in, because if for some reason you take out the isCoach prop later on, you won't have a default value to fall back on and nothing is gonna be displayed.